dbicdh/PostgreSQL/upgrade/92-93/001-auto.sql

```diff
@@ -0,0 +1,12 @@
+-- Convert schema '/hdd/openqa-devel/repos/openQA/script/../dbicdh/_source/deploy/92/001-auto.yml' to '/hdd/openqa-devel/repos/openQA/script/../dbicdh/_source/deploy/93/001-auto.yml':;
+
+;
+BEGIN;
+
+;
+ALTER TABLE jobs ADD COLUMN is_archived boolean DEFAULT '0' NOT NULL;
```

Comment:
"is_archived" sounds more like a method. I think in other cases of boolean columns we did not add the "is_" prefix.

Reply:
DBIx maps it to a method so I thought it would be a good fit. We already do the same for the boolean columns within the users table (`is_operator`, `is_admin`).

Reply:
To be fair, there are also other boolean columns *not* using that prefix. I don't really care but would postpone polishing like this until we decide that we actually want to merge the spike solution.

Reply:
sure, not important right now. Note for later :)

Reply:
I'd also just call this column `archived`.